muikku-timed-notifications-plugin/src/main/java/fi/otavanopisto/muikku/plugins/timed/notifications/strategies/AbstractTimedNotificationStrategy.java

```diff
@@ -81,8 +87,26 @@ private void startTimer(long duration) {
   }
 
   protected List<OrganizationEntity> getActiveOrganizations() {
-    // TODO: Organization-based activation of (individual?) notifiers
-    return organizationEntityController.listUnarchived();
+    List<OrganizationEntity> result = new ArrayList<>();
+    
+    if (StringUtils.isNotBlank(ENABLED_ORGANIZATIONS)) {
+      String[] organizationIdentifiers = StringUtils.split(ENABLED_ORGANIZATIONS, ",");
+
+      for (String organizationIdentifierStr : organizationIdentifiers) {
+        if (StringUtils.isNotBlank(organizationIdentifierStr)) {
+          SchoolDataIdentifier identifier = SchoolDataIdentifier.fromId(organizationIdentifierStr);
+          
+          if (identifier != null) {
+            OrganizationEntity organizationEntity = organizationEntityController.findByDataSourceAndIdentifier(identifier.getDataSource(), identifier.getIdentifier());
```

Comment:
Probably needs to be replaced with findByDataSourceAndIdentifierAndArchived from the respective branch